retriever/lib/get_opts.py

```diff
@@ -70,8 +70,8 @@
 license_parser.add_argument('dataset', help='dataset name', nargs='?', default=None, choices=script_list + [None])
 new_parser.add_argument('filename', help='new script filename')
 edit_json_parser.add_argument('dataset', help='dataset name', choices=json_list)
-reset_parser.add_argument('scope', help='things to reset: all, scripts, data, or connections',
-                          choices=['all', 'scripts', 'data', 'connections'])
+reset_parser.add_argument('scope', help='things to reset: all, scripts, data, or connections').completer = \
+    ChoicesCompleter(script_list + ['all', 'scripts', 'data', 'connections'])
```

Comment:
Remove the `connections`, This is part is no longer supported.
Also line `https://github.com/weecology/retriever/pull/1242/files#diff-09b72f49b55e7226a62ae92a8a825f82L97`